lib/ui/viewer/file_details/file_details_divider_widget.dart

```diff
@@ -0,0 +1,17 @@
+import "package:flutter/material.dart";
+import "package:photos/ui/components/divider_widget.dart";
+
+class FileDetialsDividerWidget extends StatelessWidget {
```

Comment:
Should we extend `DividerWidget` and add optional padding parameter in that. That way we can void defining this (& similar  widgets) which just add padding